### ISSUE TITLE
-Djenkins.install.runSetupWizard=false not apply. /etc/default/jenkins does not work after Jenkins version 2.332.1

### ISSUE THREAD

**Issue by patsevanton** [geerlingguy/ansible-role-jenkins#357]
Hello! Thanks for role jenkins

/etc/default/jenkins does not work after Jenkins version 2.332.1, which relies on systemd rather than the init system 
https://www.jenkins.io/doc/upgrade-guide/2.332/

**Inventory**
```
all:
  children:
    jenkins:
      hosts:
        "jenkins":
          ansible_host: "xxx"
  vars:
    jenkins_hostname: "xxx"
    jenkins_plugins:
      - cloudbees-folder
```

**playbook.yml**
```
- hosts: jenkins
  become: true
  roles:
    - role: robertdebock.java
    - role: geerlingguy.jenkins
```

**Output run playbook**
```
TASK [geerlingguy.jenkins : Modify variables in init file.] ************************************************************************************
task path: /home/user/.ansible/roles/geerlingguy.jenkins/tasks/settings.yml:13
changed: [jenkins] => (item={'option': 'JENKINS_ARGS', 'value': '--prefix='}) => changed=true 
  ansible_loop_var: item
  backup: ''
  item:
    option: JENKINS_ARGS
    value: --prefix=
  msg: line added
changed: [jenkins] => (item={'option': 'JAVA_ARGS', 'value': '-Djenkins.install.runSetupWizard=false'}) => changed=true 
  ansible_loop_var: item
  backup: ''
  item:
    option: JAVA_ARGS
    value: -Djenkins.install.runSetupWizard=false
  msg: line added
```

**cat /etc/default/jenkins | grep -v "#" | grep -v "^$"**
```
NAME=jenkins
JAVA_ARGS="-Djava.awt.headless=true"
JAVA_ARGS="$JAVA_ARGS -Djenkins.install.runSetupWizard=false"
PIDFILE=/var/run/$NAME/$NAME.pid
JENKINS_USER=$NAME
JENKINS_GROUP=$NAME
JENKINS_WAR=/usr/share/java/$NAME.war
JENKINS_HOME=/var/lib/jenkins
RUN_STANDALONE=true
JENKINS_LOG=/var/log/$NAME/$NAME.log
JENKINS_ENABLE_ACCESS_LOG="no"
MAXOPENFILES=8192
HTTP_PORT=8080
PREFIX=/$NAME
JENKINS_ARGS="--webroot=/var/cache/$NAME/war --httpPort=$HTTP_PORT"
JENKINS_ARGS="$JENKINS_ARGS --prefix="
```

**sudo systemctl cat jenkins | grep -v "#" | grep -v "^$"**
```
[Unit]
Description=Jenkins Continuous Integration Server
Requires=network.target
After=network.target
[Service]
Type=notify
NotifyAccess=main
ExecStart=/usr/bin/jenkins
Restart=on-failure
SuccessExitStatus=143
User=jenkins
Group=jenkins
Environment="JENKINS_HOME=/var/lib/jenkins"
WorkingDirectory=/var/lib/jenkins
Environment="JENKINS_WEBROOT=%C/jenkins/war"
Environment="JAVA_OPTS=-Djava.awt.headless=true"
Environment="JENKINS_PORT=8080"
[Install]
WantedBy=multi-user.target
```

![image](https://user-images.githubusercontent.com/10828883/180182424-524b2f9d-3f4b-4288-8287-5ca88c620038.png)

**Comment by patsevanton** [comment]
```
- hosts: jenkins
  become: true
  roles:
    - role: robertdebock.java
    - role: geerlingguy.jenkins
    - role: systemli.letsencrypt
    - role: geerlingguy.nginx
  post_tasks:
    - name: Jenkins Skip startUp for MI. below works for 2.332.1 or latest
      lineinfile:
        dest=/usr/lib/systemd/system/jenkins.service
        regexp='^Environment="JAVA_OPTS=-Djava.awt.headless=true'
        line='Environment="JAVA_OPTS=-Djava.awt.headless=true -Djenkins.install.runSetupWizard=false"'
      when: jenkins_version is not defined
      register: jenkins_service
    
    - name: Restart service jenkins, also issue daemon-reload to pick up config changes
      ansible.builtin.systemd:
        state: restarted
        daemon_reload: yes
        name: jenkins
      when: jenkins_service.changed
```

**Comment by MatthewRFennell** [comment]
This looks to me like a possible duplicate of #351

**Comment by geerlingguy** [comment]
Yeah, let's keep the discussion focused in #351 and at least initial fixes in #354.
